apps/dashboard/app/views/batch_connect/session_contexts/_form.html.erb

```diff
@@ -1,4 +1,6 @@
 
+<%= OodAppkit.markdown.render(@app.form_header).html_safe %>
```

Comment:
Should it be `html_safe`? I feel like that's a way to inject `<script>` tags in an odd way.

Links I think maybe OK - but it seem to me that we should sanitize it in some way to be sure at least `script`s don't work.

https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html

Reply:
I did test `<script>` tags, and they seemed to be sanitized away, but I'll take another look at it.

Reply:
Thanks for looking into it, I didn't try myself, I just think about it when i see `html_safe`.

Reply:
Replaced html_safe with sanitize. Using the defaults for now, which seemed to even sanitize away `div` elements and id tags used in the tests. I haven't looked into whether the permitted tags list needs to be customized or not.

Reply:
Realistically I'm only worried about `script` tags.